api/v1alpha1/capp_types.go

```diff
@@ -52,6 +54,22 @@ type CappSpec struct {
 
 	// LogSpec defines the configuration for shipping Capp logs.
 	LogSpec LogSpec `json:"logSpec,omitempty"`
+
+	// VolumesSpec defines the volumes specification for the Capp.
+	VolumesSpec VolumesSpec `json:"volumesSpec,omitempty"`
+}
+
+// VolumesSpec defines the volumes specification for the Capp.
+type VolumesSpec struct {
+	NfsVolumes []NfsVolume `json:"nfsVolumes,omitempty"`
+}
+
+// NfsVolume defines the NFS volume specification for the Capp.
+type NfsVolume struct {
+	Server   string              `json:"server"`
+	Path     string              `json:"path"`
+	Name     string              `json:"name"`
+	Capacity corev1.ResourceList `json:"capacity"`
```

Comment:
I don't want to ask for Access mode because it won't work with 'ReadWriteOne'

Reply:
I think it would be better to use the Spec directly and then limit the options with a webhook since it would allow us to keep consistency between the NFSPVC and Capp easily